ansible/roles/operator-pipeline/templates/openshift/tasks/publish-to-index.yml

```diff
@@ -93,20 +71,28 @@ spec:
             *)
                 echo "Publishing bundle to an index is a NOOP for dev and qa environments at this time."
                 echo -n "success" | tee "$(results.status.path)"
-                # output dummy/test values for signing purposes
-                echo -n "$(params.bundle_pullspec)" | tee "$(results.manifest_list_digests.path)"
                 exit 0
             ;;
         esac
 
-        # DO NOT use `--verbose` to avoid auth headers appearing in logs
-        index \
-          --iib-url "$(params.iib_url)" \
-          --from-index $FROM_INDEX \
-          --indices $(params.index_images) \
-          --bundle-pullspec "$(params.bundle_pullspec)" \
-          --output manifest-list-digests.txt
+        echo "FROM_INDEX: $FROM_INDEX"
+        echo "Copying index images to published repos..."
+
+        TEMP_IMAGES=$(echo $(params.index_image_paths) | tr "," " ")
 
+        for i in $TEMP_IMAGES
+        do
+          SRC_IMAGE=$(echo $i | awk -F '+' '{print $2}')
+          echo "Source image: $SRC_IMAGE"
+          VERSION=$(echo $i | awk -F '+' '{print $1}' | awk -F ':' '{print $2}')
+          DEST_IMAGE="${FROM_INDEX}:${VERSION}"
+          echo "Dest image: $DEST_IMAGE"
 
-        echo -n "success" | tee "$(results.status.path)"
-        cat manifest-list-digests.txt | tee "$(results.manifest_list_digests.path)"
+          skopeo \
+            --command-timeout 300s copy \
+            --format v2s2 --all \
```

Comment:
Is th `--format` necessary here? I have seen cases when images were converted from oci to docker schema and digest have been changed. Also, skopeo can fail if this option is used and the input image can't be converted.

Reply:
This is what IIB is currently using according to their logs, so I'm just trying to preserve existing behaviors.